packages/python/accern_xyme/accern_xyme.py

```diff
@@ -2375,6 +2400,32 @@ def upload_zip(self, source: Union[str, io.BytesIO]) -> List['BlobHandle']:
             for blob_uri in files
         ]
 
+    def upload_sklike_model(
+            self,
+            source: Union[str, io.BytesIO],
+            xcols: List[str],
+            is_clf: bool,
+            model_name: str,
+            maybe_classes: Optional[List[str]],
+            maybe_range: Optional[Tuple[Optional[float], Optional[float]]],
+            full_init: bool = True) -> UploadFilesResponse:
+        try:
+            if isinstance(source, str) or not hasattr(source, "read"):
+                with open(f"{source}", "rb") as fin:
+                    self._upload_file(fin, ext="pkl")
+            else:
+                self._upload_file(source, ext="pkl")
```

Comment:
where does upload_file set the filename?

Reply:
we always save the file as 'model.pkl' at the backend